desc/profiles.py

```diff
@@ -197,13 +197,13 @@ def to_mtanh(
             **kwargs,
```

Comment:
Do you need to use the `__call__` method? In most places we use the `compute` method which allows you to create the grid outside and pass it in.

If you want to keep this, I think we can just always make the grid jitable here, since we know it only has radial resolution and none of the profiles need to know the indexing stuff.

Reply:
I'll just make the grid outside. It's probably best to not always make it jitable because that will remove the nodes of the grid on the magnetic axis, which can cause issues that are hard to debug.